main.tf

```diff
@@ -182,6 +182,24 @@ resource "spacelift_policy_attachment" "trigger_administrative" {
   stack_id  = data.spacelift_current_stack.administrative[0].id
 }
 
+# global administrative push policy that updates branch tracking in the admin stack
+resource "spacelift_policy" "push_administrative" {
```

Comment:
cc: @danjbh @aknysh

Reply:
The downside to that would be we'll have to go to each client and make sure they add this policy `policies_by_id_enabled`, but we'll have to bump the module version at the same time anyhow so perhaps it's not that big of a deal.

I'm good with whatever you guys want to do.

@nitrocode I think I'll be handing this ticket off to you at this point since I'm transitioning to a new project. I'll send the tickets your way.

Reply:
cc: @aknysh thoughts on this?

Reply:
After discussing with @aknysh on slack, we decided that this PR is the best way to go here for existing clients. 

Separately for our new clients and existing ones with multiple admin stacks, we can have our original admin stack (or separate one for just policies) create the policies and then use policies_by_id_enabled for the other admin stacks